ddl/ddl_api.go

```diff
@@ -2060,10 +2062,6 @@ func (d *ddl) AddTablePartitions(ctx sessionctx.Context, ident ast.Ident, spec *
 	if meta.GetPartitionInfo() == nil {
 		return errors.Trace(ErrPartitionMgmtOnNonpartitioned)
 	}
-	// We don't support add hash type partition now.
-	if meta.Partition.Type == model.PartitionTypeHash {
```

Comment:
I guess it means `alter table add partition ...` on a hash partitioned table... and we still don't support that operation?

Reply:
We still don't support that operation, the check is moved into `buildPartitionInfo`.

Reply:
ok, I see it